tests/bsim/bluetooth/audio/src/bap_unicast_client_test.c

```diff
@@ -106,6 +102,16 @@ static void stream_started(struct bt_bap_stream *stream)
 {
 	printk("Started stream %p\n", stream);
 
+	if (stream_tx_can_send(stream)) {
```

Comment:
question: Do I understand it correctly that when `stream_tx_can_send` is false we do not actually start streaming for this `stream`?
If my understanding is correct then probably we shouldn't set the `flag_stream_started`

Reply:
Streams are unidirectional. For RX-only streams `stream_tx_can_send` will return false, but the stream is still started in the sense that the ASE is in the streaming state